Library/Homebrew/cmd/orphaned.rb

```diff
@@ -0,0 +1,13 @@
+#:  * `orphaned`:
+#:    Lists kegs that were installed as dependencies of other kegs, but are no
```

Comment:
Good catch @maxim-belkin. That's more consistent with the other Homebrew commands.